stable/postgresql/templates/metrics-deployment.yaml

```diff
@@ -0,0 +1,75 @@
+{{- if .Values.metrics.enabled }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "postgresql.fullname" . }}-metrics
+  labels:
+    app: {{ template "postgresql.name" . }}
+    chart: {{ template "postgresql.chart" . }}
+    release: {{ .Release.Name | quote}}
+    heritage: {{ .Release.Service | quote }}
+spec:
+  template:
+    metadata:
+      labels:
+        release: {{ .Release.Name | quote }}
+        app: {{ template "postgresql.name" . }}
+        chart: {{ template "postgresql.chart" . }}
+        role: metrics
+      {{- if .Values.metrics.podLabels }}
+{{ toYaml .Values.metrics.podLabels | indent 8 }}
+      {{- end }}
+      {{- if .Values.metrics.podAnnotations }}
+      annotations:
+{{ toYaml .Values.metrics.podAnnotations | indent 8 }}
+      {{- end }}
+    spec:
+      {{- if .Values.metrics.image.pullSecrets }}
+      imagePullSecrets:
+        {{- range .Values.metrics.image.pullSecrets }}
+        - name: {{ . }}
+        {{- end}}
+      {{- end}}
+      {{- if .Values.metrics.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.metrics.nodeSelector | indent 8 }}
+      {{- end }}
+      {{- if .Values.metrics.tolerations }}
+      tolerations:
+{{ toYaml .Values.metrics.tolerations | indent 8 }}
+      {{- end }}
+      containers:
+      - name: metrics
+        image: {{ template "metrics.image" . }}
+        imagePullPolicy: {{ .Values.metrics.image.pullPolicy | quote }}
+        env:
+        - name: DATA_SOURCE_NAME
```

Comment:
I think the metrics configuration is outdated. @javsalgar implemented this changes:

https://github.com/bitnami/charts/pull/875